### PR TITLE
[Split_2]add nuspec

### PR DESCRIPTION
**Pull request by vicancy**

### CODE REVIEW

File: plugins/Microsoft.DocAsCode.Build.MemberLevelManagedReference/docfx.plugins.config [hunk 1]
```diff
@@ -0,0 +1,23 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
```
Comment (vwxyzh) [inline plugins/Microsoft.DocAsCode.Build.MemberLevelManagedReference/docfx.plugins.config]: why add this file?

**Comment by superyyrrzz** [comment]
:shipit:

**Comment by qinezh** [comment]
:shipit:

**Comment by hellosnow** [comment]
:shipit:

**Comment by superyyrrzz** [comment]
Come up with a question... For OPS, if this package is `-t`'s last parameter, template in DOCS.MSFT will be overwritten...

**Comment by vicancy** [comment]
Nice catch, need to exclude it from nuspec

File: plugins/Microsoft.DocAsCode.Build.MemberLevelManagedReference/memberpage.nuspec [hunk 2]
```diff
@@ -0,0 +1,21 @@
+﻿<?xml version="1.0"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2011/10/nuspec.xsd">
+  <metadata>
+    <id>Microsoft.DocAsCode.Build.MemberLevelManagedReference</id>
```
Comment (superyyrrzz) [inline plugins/Microsoft.DocAsCode.Build.MemberLevelManagedReference/memberpage.nuspec]: >Microsoft.DocAsCode.Build.MemberLevelManagedReference [](start = 8, length = 53)

rename id and title?

File: plugins/Microsoft.DocAsCode.Build.MemberLevelManagedReference/memberpage.plugins.nuspec [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+    <id>Microsoft.DocAsCode.Build.MemberLevelManagedReference</id>
+    <version>1.0.0</version>
+    <title>Microsoft.DocAsCode.Build.MemberLevelManagedReference</title>
+    <authors>Microsoft</authors>
+    <owners>Microsoft</owners>
+    <requireLicenseAcceptance>true</requireLicenseAcceptance>
+    <licenseUrl>https://github.com/dotnet/docfx/blob/dev/LICENSE</licenseUrl>
+    <projectUrl>https://github.com/dotnet/docfx</projectUrl>
+    <description>This is a plugin for DocFX to split ManagedReference classes into member level pages. Members with the same overload name are grouped into one page, members with no overload are splited to one page per member. This package contains the plugin for ManagedReference document processor only.</description>
+    <releaseNotes>Add custom templates</releaseNotes>
```
Comment (superyyrrzz) [inline plugins/Microsoft.DocAsCode.Build.MemberLevelManagedReference/memberpage.plugins.nuspec]: >Add custom templates [](start = 18, length = 20)

update

**Comment by superyyrrzz** [comment]
:shipit:

**Pull request merged**